doc/nrf/ug_nrf52.rst

```diff
@@ -122,6 +122,20 @@ In addition, you can run the :ref:`zephyr:bluetooth-samples` that are included f
 
 For available libraries, see :ref:`lib_bluetooth_services` (|NCS|) and :ref:`zephyr:bluetooth_api` (Zephyr).
 
+Bluetooth mesh
+==============
+
+Bluetooth mesh is a low-power mesh networking standard based on Bluetooth Low Energy, optimized for creating large-scale device networks.
```

Comment:
I don't think there is a standard actually called "Bluetooth mesh"? Perhaps replace with "The Bluetooth Mesh Profile" in this paragraph, as it specifically mentions "is a [...] standard"?

Reply:
Can I use solution or technology instead of standard? I see SIG is using networking solution and/or technology when talking about Bluetooth mesh networking.

Reply:
Solution would work